src/examples/mfem_mortar_lm_patch.cpp

```diff
@@ -369,23 +381,45 @@ int main( int argc, char** argv )
   mfem::Vector g;
   tribol::getMfemGap(coupling_scheme_id, g);
 
-  // Apply a transpose prologation operator on the submesh: maps dofs stored in
-  // dual field g to tdofs stored in G for parallel solution of the linear
-  // system.
+  // Apply a restriction operator on the submesh: maps dofs stored in g to tdofs
+  // stored in G for parallel solution of the linear system.
+  //
+  // NOTE: gap is a dual field, but we still apply R here because tribol stores
+  // this like a ParGridFunction: shared DOFs have the same (summed) value on
+  // all ranks
   {
     auto& G = B_blk.GetBlock(1);
     auto& R_submesh = *tribol::getMfemPressure(coupling_scheme_id)
-      .ParFESpace()->GetProlongationMatrix();
-    R_submesh.MultTranspose(g, G);
+      .ParFESpace()->GetRestrictionMatrix();
```

Comment:
This looks like a bug fix?

Reply:
Correct, the added comment above explains why the fix is necessary